src/containers/checkout/checkout-form/checkout-form.js

```diff
@@ -50,57 +42,55 @@ import {
 
 const { pathToUserAgreement, pathToCart } = routes;
 
-const CheckoutForm = ({ language, isLightTheme, currency, cartItems, deliveryType }) => {
+const CheckoutForm = ({ isLightTheme, currency, cartItems, deliveryType }) => {
   const styles = useStyles({
     isLightTheme
   });
   const currencySign = getCurrencySign(currency);
   const userData = useSelector(({ User }) => User.userData);
-  const { t } = useTranslation();
+  const { t, i18n } = useTranslation();
+  const getCheckoutPayment = t('checkout.checkoutPayment', { returnObjects: true });
+  const language = i18n.language === 'ua' ? 0 : 1;
 
   const dispatch = useDispatch();
   const totalPriceToPay = cartItems.reduce(
     (previousValue, currentValue) =>
       previousValue + calcPriceForCart(currentValue, currency, currentValue.quantity),
     0
   );
-  const consentLink =
-    language === LANGUAGES_LIST[0].value ? (
-      <div className={styles.consentMessage}>
+  const consentLink = (
+    <div className={styles.consentMessage}>
+      {' '}
+      {t('checkout.checkoutAdditionalInfo.consent.0')}
+      <Link
+        className={styles.consentLink}
+        to={pathToUserAgreement}
+        target='_blank'
+        rel='noreferrer'
+      >
         {' '}
-        {CHECKOUT_ADDITIONAL_INFORMATION[language].consent[0]}
-        <Link
-          className={styles.consentLink}
-          to={pathToUserAgreement}
-          target='_blank'
-          rel='noreferrer'
-        >
-          {' '}
-          {CHECKOUT_ADDITIONAL_INFORMATION[language].consent[1]}{' '}
-        </Link>
-      </div>
-    ) : (
-      ''
-    );
+        {t('checkout.checkoutAdditionalInfo.consent.1')}{' '}
+      </Link>
+    </div>
+  );
 
   const { dirty, values, handleSubmit, handleChange, setFieldValue, touched, errors, resetForm } =
     useFormik({
       validationSchema: validationSchema(deliveryType, t),
       initialValues,
 
       onSubmit: (data) => {
-        data.paymentMethod === CHECKOUT_PAYMENT[language].card
-          ? dispatch(addPaymentMethod(CHECKOUT_PAYMENT[language].card)) &&
+        data.paymentMethod === t('checkout.checkoutPayment.card')
```

Comment:
Translations are only for translating values, not for the ternaries or maps etc. I think we need to refactor this and not use translations in logic elements.